frontend/src/metabase/collections/containers/CollectionSidebar/CollectionSidebar.jsx

```diff
@@ -79,90 +72,46 @@ class CollectionSidebar extends React.Component {
     return (
       <React.Fragment>
         <ToggleMobileSidebarIcon onClick={handleToggleMobileSidebar} />
+
         <Collection.Loader id="root">
-          {({ collection: root }) => (
-            <Box mb={1} mt={2}>
-              <CollectionDropTarget collection={root}>
-                {({ highlighted, hovered }) => (
-                  <CollectionLink
-                    to={Urls.collection({ id: "root" })}
-                    selected={isRoot}
-                    highlighted={highlighted}
-                    hovered={hovered}
-                  >
-                    {t`Our analytics`}
-                  </CollectionLink>
-                )}
-              </CollectionDropTarget>
-            </Box>
-          )}
+          {({ collection: root }) => <Header isRoot={isRoot} root={root} />}
```

Comment:
Was a bit confused seeing a `Header` here, thought it's related to mobile UI improvements you're working on; however, it's still a link to "Our analytics". Should we maybe just call it smth like `RootCollectionLink` to avoid confusion?

Reply:
Thank you very much @kulyk, I will apply all your naming suggestions and let's move forward 🙇